File: app/src/main/java/com/thoughtworks/pafsilva/testautomationworkshop/retrofit/APIEndpoints.java
package com.thoughtworks.pafsilva.testautomationworkshop.retrofit;

import com.thoughtworks.pafsilva.testautomationworkshop.model.User;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface APIEndpoints {

    @GET("/users")
    Call<List<User>> getUser(@Query("email") String email, @Query("password") String password);
}

File: app/src/main/java/com/thoughtworks/pafsilva/testautomationworkshop/retrofit/APIEndpoints.kt
package com.thoughtworks.pafsilva.testautomationworkshop.retrofit

import com.thoughtworks.pafsilva.testautomationworkshop.model.User

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface APIEndpoints {

    @GET("/users")
    fun getUser(@Query("email") email: String, @Query("password") password: String): Call<List<User>>
}